recipes/recipes/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for recipes project.
+
+Generated by 'django-admin startproject' using Django 3.1.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'y4o&sasd(&%^01i+&wuo8rznlw&_gi6=hsl&45(4mvbx4609uv'
```

Comment:
:scream:  Usually best to put "FakeKey" here or something so folks don't freak out. Now TripActions will start making our recipes.

Reply:
Haha! Oops! :)